content/configuration/artifacts.md

```diff
@@ -0,0 +1,183 @@
+
+It is also possible to specify a build directory, which works in conjunction with the Docker file path: 
+
+``` yaml
+tasks:
+    images:
+        build:
+            services:
+                web:
+                    steps:
+                        - docker_file_path: ./Buildfile
+                          build_directory: ./docker
+                        - docker_file_path: ./Dockerfile
+                          build_directory: ./docker
+                          image: ${IMAGE_NAME} 
+```
+
+In this configuration the Docker file paths will be `./docker/Buildfile` and `./docker/Dockerfile` respectively.
+
+## Using Artifacts
+
+To enable artifacts, a `write_artifacts` and `read_artifacts` section needs to be added to separate build steps as follows:
+
+``` yaml
+tasks:
+    images:
+        build:
+            services:
+                web:
+                    steps:
+                        - docker_file_path: ./Buildfile
+                          write_artifacts:
+                              - name: built-files-artifact
+                                path: /app/built-files
+                        - docker_file_path: ./Dockerfile
+                          image: ${IMAGE_NAME}
+                          read_artifacts:
+                              - name: built-files-artifact
+                                path: /built-files
+```
+
+The `write_artifacts` section in the first build step creates a write artifact with name "built-files-artifact". It then specifies a path of "/app/built-files" which represents the location where the Docker file `Buildfile` will place the results of the build process that it intends to share. 
+
+The `read_artifacts` section in the second build step creates a corresponding read artifact by referencing the same name as the write artifact i.e. "built-files-artifact". It then specifies a path of "/built-files" which represents the location where the Docker file `Dockerfile` can expect to find the results of the build process that have been shared.
+
+### Docker File for First Build Step
+
+The following is a minimal example Docker file to demonstrate the interaction between the artifact and Docker in the first build step:
+
+> **Buildfile**
+```
+FROM nginx
+
+RUN mkdir -p /app/built-files && touch /app/built-files/test.txt && echo "test1234" > /app/built-files/test.txt
+```
+
+This simply creates a directory that matches the path specified in the write artifact i.e "/app/built-files", and then creates a file within the directory to populate the artifact.
+
+{{< note title="Note" >}}
+In practice the contents of the "/app/built-files" directory would be the result of a build process.
+{{< /note >}}
+
+### Docker File for Second Build Step
+
+The following is a minimal example Docker file to demonstrate the interaction between the artifact and Docker in the second build step:
+
+> **Dockerfile**
+```
+FROM nginx
+
+COPY . /usr/share/nginx/html
```

Comment:
I'd remove this line to prevent any confusion, and the fact that with such example, your `built-files` folder will be copied twice. If you want to demonstrate the fact that you still have the source code available, maybe use something like the following:
```
COPY ./etc/nginx /etc/nginx
```